### ISSUE TITLE
[Issue] Biscuit QC doesn't run / parallel error

### ISSUE THREAD

**Issue by semenko** [huishenlab/biscuit#32]
I don't think the Biscuit QC script runs as is -- I tried patching this in #31 -- would you consider merging?

(Would love an updated bioconda release with your recent awesome changes as well 😄 )

**Comment by jamorrison** [comment]
Hi @semenko,

Thanks for your issue and bug fix. Just to confirm, the QC script worked for you with `-j6`, just not as advertised with 6 cores. If so, then I'll go ahead and merge your PR.

As for the updated release, you can expect a release sometime this month. I have a major change that I'm working through for the `epiread` subcommand and once that's ready (it's almost there), I'll go ahead and craft a release and push that to bioconda.

**Comment by semenko** [comment]
(I wonder if this is the same issue PR #29 alludes to. Perhaps it makes sense to remove the GNU parallel dependency entirely, and just spawn those jobs to the background, plus a simple `wait` in QC.sh)

**Comment by jamorrison** [comment]
I confirmed this issue and merged your PR. I'd guess that was the issue alluded to in #29 also.

I've found this portion of `QC.sh` (depth stats) to be horribly slow and it easily takes the largest fraction of time when running the whole script. I've gone back and forth on whether to farm this out to another tool (`mosdepth` or something similar) or rework the current code to improve runtime. Ideally, either option would avoid using GNU parallel.
